doc/source/user_guide/io.rst

```diff
@@ -2864,6 +2864,16 @@ of sheet names can simply be passed to ``read_excel`` with no loss in performanc
     data = pd.read_excel('path_to_file.xls', ['Sheet1', 'Sheet2'],
                          index_col=None, na_values=['NA'])
 
+If control is needed over how a file is read, an ``xlrd`` workbook 
+created with the desired keyword arguments can be passed to ``ExcelFile``.
```

Comment:
May be it's just me, but I don't find this sentence very easy to read.

I'd rephrase to something like: "`pandas.ExcelFile` can also be called with an `xlrd.open_workbook` as a parameter. This lets the user have control on how the workbook is opened, for example with `on_demand=True`". Or something similar.

CC: @WillAyd @TomAugspurger

Reply:
@datapythonista's sounds good, though I might change "an xldd.open_workbook" to whatever type of object that returns.

Reply:
I agree. I just committed the suggested changes.